kratos/geometries/hexahedra_3d_27.h

```diff
@@ -373,11 +391,90 @@ template<class TPointType> class Hexahedra3D27 : public Geometry<TPointType>
      * Operations
      */
 
+    /**
+     * @brief Creates a new geometry pointer
+     * @param ThisPoints the nodes of the new geometry
+     * @return Pointer to the new geometry
+     */
     typename BaseType::Pointer Create( PointsArrayType const& ThisPoints ) const override
     {
         return typename BaseType::Pointer( new Hexahedra3D27( ThisPoints ) );
     }
 
+    /**
+     * @brief Creates a new geometry pointer
+     * @param NewGeometryId the ID of the new geometry
+     * @param ThisPoints the nodes of the new geometry
+     * @return Pointer to the new geometry
+     */
+    typename BaseType::Pointer Create(
+        IndexType NewId,  
+        PointsArrayType const& ThisPoints
```

Comment:
```suggestion
        PointsArrayType const& rThisPoints
```

Do check for other places where this change is necessary.